src/main/java/org/mdcfg/provider/MdcProvider.java

```diff
@@ -588,6 +586,60 @@ public <K, V> Map<K, V> getMap(MdcContext context, String key, Function<String,
         return Collections.emptyMap();
     }
 
+    /**
+     * Read compound property and return result as Tree (based on maps).
+     *
+     * @param context reading context {@link MdcContext}.
+     * @param key property name.
+     * @return {@code Map} of property values.
+     * @throws MdcException in case property not found.
+     */
```

Comment:
Would it be more clear to name it getPropertyMap? Or how do we call a pair of key:value in mdc? entry? configEntry? property?